packages/openmrs/src/Utils.js

```diff
@@ -21,18 +21,52 @@ export function request(state, method, path, data, params) {
 
   const options = {
     body: data,
-
     headers: {
       ...headers,
       'content-type': 'application/json',
     },
-
     query: params,
-
     parseAs: 'json',
   };
 
   const url = `${instanceUrl}${path}`;
 
-  return commonRequest(method, url, options).then(response => logResponse(response));
+  return autoFetchRequest(method, url, options);
```

Comment:
I don't think we need `request` and `autoFetchRequest` - it doesn't seem to be a helpful division of functionality. I'd just move the autofetch stuff inside the main function.

Reply:
Removed autoFetchRequest function